src/Objs/defaultPageEditingConfig.ts

```diff
@@ -14,13 +14,21 @@ export const defaultPageEditingConfigAttributes = {
     title: 'Should this page be indexed?',
     description: 'If not, search engines will ignore this page. Default: Yes',
   },
+  layoutShowFooter: {
+    title: 'Show layout footer widgets?',
+  },
 }
 
 export const defaultPageInitialContent = {
   robotsIndex: true,
 } as const
 
 export const defaultPagePropertiesGroups = [
+  {
+    title: 'Layout widgets',
```

Comment:
* [ ] FYI I think "widgets" in the title will confuse editors. For an editor, the term widget most frequently refers to the WYSIWYG page elements they can D&D. Maybe just "Layout", "Layout elements", "Site layout"?